pkg/engine/engine.go

```diff
@@ -224,8 +224,13 @@ func (e *Engine) DoBuild(ctx context.Context, comp *api.Composition, basesrc str
 
 			ow.Infow("performing build for groups", "plan", plan, "groups", grpids, "builder", builder)
 
+			id, err := uuid.NewUUID()
```

Comment:
@coryschwartz this is actually affecting the names we generate for containers/images.